domains/louis.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A portofolio",
+    "repo": "https://github.com/louloudev59/portofolio",
+    "owner": {
+        "username": "<louloudev59>",
```

Comment:
Username should not contain the <>, that's just to show it is a required property.

Reply:
Oh ok, thanks